docker/Dockerfile

```diff
@@ -8,3 +8,7 @@ COPY scripts/* /scripts
 # 11.0.13 (https://github.com/haiwen/seafobj/pull/77/commits/1e38fbc1134a70c64db04c6de9f9f5d62517fb62)
 # RUN sed -i 's/addressinng_style/addressing_style/g' /opt/seafile/seafile-pro-server-11.0.13/seahub/thirdpart/seafobj/backends/s3.py
 # fixed with 11.0.14
+
+# Remove "--logfile" argument from start_seafevents()
+# This forces seafevents to log to stdout
+RUN sed -i 's/--logfile \${TOPDIR}\/logs\/seafevents\.log //g' /opt/seafile/seafile-pro-server-11.0.14/seafile-monitor.sh
```

Comment:
This workaround has the side-effect that the log files `index.,log` and `seahub_email_sender.log` are created inside `opt/seafile/seafile-server-latest`.

Source: https://github.com/haiwen/seafevents/blob/47ab626cd88ac7c18833d8ec5d153974107bf649/tasks/index_updater.py#L70